lib/cisco_node_utils/cmd_ref/bgp_neighbor.yaml

```diff
@@ -8,7 +8,7 @@ _template:
   context:
     - "router bgp <asnum>"
     - "(?)vrf <vrf>"
-    - "neighbor <nbr>"
+    - "(?)neighbor <nbr>"
```

Comment:
I reverted the yaml.  It was only failing on my "old" version of bgp_neighbor.yaml because I had not changed the all_neighbors entry.